Vungle/VungleRouter.m

```diff
@@ -536,6 +536,10 @@ - (void)vungleWillCloseAdWithViewInfo:(VungleViewInfo *)info placementID:(NSStri
     }
 
     if(targetDelegate) {
+        if ([info.didDownload isEqual:@YES]) {
+            [targetDelegate vungleAdWasTapped];
+        }
+
```

Comment:
Revert previous remove vungeAdWasTapped operation, please see below screenshot which is previous modification to remove vungeAdWasTapped:
<img width="1286" alt="Screen Shot 2019-08-28 at 3 38 17 PM" src="https://user-images.githubusercontent.com/37288907/63835482-1d9d9280-c9aa-11e9-837f-10bfb4ac1f2d.png">